src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -61,15 +62,21 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
   onPressPrevDay,
   onPressNextDay,
 }) => {
+  const { i18nt } = useTranslate();
+  const formatTime =
+    lastTransactionTime !== null
+      ? { distributionTime: format(lastTransactionTime, "h:mma") }
+      : { distributionTime: "-" };
+
   return (
     <View style={styles.appHeaderWrapper}>
-      <AppText style={styles.smallText}>You distributed</AppText>
+      <AppText style={styles.smallText}>
+        {i18nt("redemptionStats", "youDistributed")}
+      </AppText>
       <AppText style={styles.statText}>{totalCount?.toLocaleString()}</AppText>
       <AppText style={styles.smallText}>
-        Last distributed at{" "}
-        {lastTransactionTime !== null
-          ? format(lastTransactionTime, "h:mma")
-          : "-"}
+        {i18nt("redemptionStats", "lastDistributedAt")}
+        {i18nt("redemptionStats", "distributedTime", undefined, formatTime)}
```

Comment:
Similar for this, we can just use one sentence:

```suggestion
        {i18nt("redemptionStats", "lastDistributedAt", undefined, formattedLastTransactionTime)}
```